Gemfile

```diff
@@ -7,24 +7,25 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-gem 'rails', '~> 5.1.4'
+gem 'faraday'
+gem 'omniauth-uaa-oauth2', github: 'cloudfoundry/omniauth-uaa-oauth2'
 gem 'puma', '~> 3.7'
+gem 'rails', '~> 5.1.4'
 gem 'uglifier', '>= 1.3.0'
-gem 'omniauth-uaa-oauth2', github: 'cloudfoundry/omniauth-uaa-oauth2'
-gem 'faraday'
 
 group :development, :test do
-  gem 'pry-byebug'
-  gem 'rspec-rails'
   gem 'capybara'
+  gem 'pry-byebug'
   gem 'rack_session_access'
+  gem 'rspec-rails'
 end
 
 group :development do
-  gem 'web-console', '>= 3.3.0'
   gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'rubocop', '~> 0.51.0', require: false
```

Comment:
We should use `govuk-lint` as a gem and `govuk-lint-ruby` as the command, because then we don't need to vendor all of it's `.rubocop.yml` and it will get updated when we pull in new versions of the gem.

You can refer to `alphagov/paas-cf` for an example:

- https://github.com/alphagov/paas-cf/blob/c11494efc27c03ad61a4a2ced9dce548315b5da2/Gemfile#L5
- https://github.com/alphagov/paas-cf/blob/c11494efc27c03ad61a4a2ced9dce548315b5da2/Gemfile#L5
- https://github.com/alphagov/paas-cf/blob/c11494efc27c03ad61a4a2ced9dce548315b5da2/.rubocop.yml

Reply:
I'm not sure if that's the best move tbh. I can see two issues with that:

1) Anyone who has an editor/plugin that does rubocop linting for them would not understand the "govuk-lint-ruby" command and would expect to find the rubocop.yml file in the root. I find it very useful to get the cop feedback as I'm typing and it would be a real shame to lose that.

2) The govuk-lint version is not a perfect fit... I had to enable the rails specific cops and had to modify the include/excludes to ensure that certain cops are excluded from some rails specific files (large blocks in environment configs are expected for one) and needed to exclude the vendor dir so travis didnt attempt to lint the entire vendor bundle.

Reply:
... just talking with @alext  ... we think we can have our cake and eat it ... *investigating*

Reply:
The README says that there's a `-R` flag to enable Rails cops:

- https://github.com/alphagov/govuk-lint#ruby

I think it's worth trying to find solutions to because people have already done the hard work of maintaining an acceptable config ❤️

Reply:
🙌  ...sorted!

Found a way to inherit the config from the govuk-lint gem so that standard `rubocop` commands work as expected and we get the best of both worlds.